index.html

```diff
@@ -0,0 +1,350 @@
+      <a href="" class="desk-abt desk-button">About</a>
+      <a href="" class="desk-contact desk-button"> Contact</a>
+      <nav class="desk-nav desk-button">
+        <img src="assets/images/Union.png" alt="M">
+      </nav>
+    </ul>
+
+  </header>
+  <section class="first-page">
+    <div class="headline">
+      <h1 class="hey">Hey There I'm Solomon</h1>
+      <h2 class="softwaredev"> I am software developer</h2>
+      <p class="headline-p">I can help you build a product , feature or website Look through some of my work
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </div>
+
+    <div class="socio-image">
+      <ul class="socila-link">
+        <li class="social"><a href="https://github.com/solog0039"><img src="./assets/socialmedia/github.png"
+              alt="Github"></a></li>
+        <li class="social"><a href="www.linkedin.com/in/solomon-kidanu-62a994232"><img
+              src="./assets/socialmedia/linkidin.png" alt="linkedin"></a></li>
+        <li class="social"><a href="##"><img src="./assets/socialmedia/tump.png" alt="tum"></a></li>
+        <li class="social"><a href="https://twitter.com/Solomon57320119"><img src="./assets/socialmedia/tiwtter.png"
+              alt="tiwtter"></a></li>
+        <li class="social"><a href="##"><img src="./assets/socialmedia/m.png" alt="M"></a></li>
+      </ul>
+      <img src="./assets/images/Group.png" alt="">
+    </div>
+  </section>
+  <section class="desk-first-page">
+    <div class="desk-headline">
+      <h1 class="desk-hey">Hey There I'm Solomon</h1>
+      <h2 class="desk-softwaredev"> I am software developer</h2>
+      <p class="desk-headline-p">I can help you build a product , feature or website Look through some of my work
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </div>
+    <ul class="socila-link">
+      <li class="social"><a href="https://github.com/solog0039"><img src="./assets/socialmedia/github.png"
+            alt="Github"></a></li>
+      <li class="social"><a href="www.linkedin.com/in/solomon-kidanu-62a994232"><img
+            src="./assets/socialmedia/linkidin.png" alt="linkedin"></a></li>
+      <li class="social"><a href="##"><img src="./assets/socialmedia/tump.png" alt="tum"></a></li>
+      <li class="social"><a href="https://twitter.com/Solomon57320119"><img src="./assets/socialmedia/tiwtter.png"
+            alt="tiwtter"></a></li>
+      <li class="social"><a href="##"><img src="./assets/socialmedia/m.png" alt="M"></a></li>
+    </ul>
+    <img src="./assets/images/Gourp1-desktop.png" alt="Image in the first page">
+  </section>
+  <section class="cards">
+    <div class="h21-line">
+      <h1 class="h21">My Recent Works</h1>
+      <hr class="line">
+    </div>
+    <img class="main-image" src="assets/images/Placeholder.png" alt="main image">
+
+    <div class="card-1">
```

Comment:
[OPTIONAL]
- [ ] The placement of the `see project` button text in the first card of your portfolio page is not correctly centered as shown in the figma template below, please can you work on fixing this before moving on to your next project :+1:  
- figma design
![spf-f](https://user-images.githubusercontent.com/89055502/236587580-cc3f3853-196f-4572-8d74-2047b5a2f872.png)
- your design
![spf-s](https://user-images.githubusercontent.com/89055502/236587588-3e9c11e6-537b-4426-872d-7b6975d19212.png)